frontend/src/components/common/PrintCSVButton.tsx

```diff
@@ -66,19 +67,19 @@ const PrintCSVButton = (): React.ReactElement => {
 
   return (
     <>
-      <IconButton
-        aria-label="Print CSV"
-        className="ghost-button"
-        icon={<Icon boxSize="32px" as={AiFillPrinter} />}
-        variant="ghost"
-        onClick={handleOpen}
-      />
+      <Tooltip label="Export to CSV">
+        <IconButton
+          icon={<Icon boxSize="32px" as={ExternalLinkIcon} />}
+          variant="tertiary"
+          onClick={handleOpen}
+        />
```

Comment:
Currently encountering a compilation error :(
<img width="937" alt="Screen Shot 2023-06-23 at 10 11 44 PM" src="https://github.com/uwblueprint/supportive-housing/assets/53386475/7ded85e6-c9fa-4675-8418-3f67197db0e1">

```suggestion
        <IconButton
          aria-label="Print CSV"
          icon={<Icon boxSize="32px" as={ExternalLinkIcon} />}
          variant="tertiary"
          onClick={handleOpen}
        />
```

Reply:
I saw the same issue, hopefully #121 will fix this

if not, the suggestion above will work :D

Reply:
I am going to put this fix in now and we can always check if it is unnecessary later on.